lib/ui/src/components/layout/mobile.tsx

```diff
@@ -186,7 +190,7 @@ class Mobile extends Component<MobileProps, MobileState> {
           </div>
           <Panel hidden={!viewMode} />
         </Panels>
-        <Bar>
+        <Bar isFullscreen={options.isFullscreen}>
```

Comment:
Rather than passing an `isFullscreen` prop to the styled component, what if we conditionally render it?

```suggestion
        {options.isFullscreen && (
          <Bar>
          // ...
        )}
```

Reply:
You're completely right, I have changed it :)